lib/providers/auth/assign_dio_headers_provider.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:io';
+
+import 'package:Medito/models/models.dart';
+import 'package:Medito/providers/providers.dart';
+import 'package:Medito/services/network/dio_client_provider.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+var assignDioHeadersProvider = FutureProvider<void>((ref) async {
+  var _authProvider = ref.read(authProvider);
```

Comment:
If the var is declared inside a function it doesn't need to be private, since the scope is already restricted.